src/App.tsx

```diff
@@ -1,61 +1,160 @@
+import React, { FormEvent, useState } from 'react';
+import Button from '@mui/material/Button';
+import SendIcon from '@mui/material/Icon';
+import Input from '@mui/material/TextField';
+import NativeSelect from '@mui/material/NativeSelect';
 import './App.scss';
+import cn from 'classnames';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+
+const ariaLabel = { 'aria-label': 'description' };
+
+function getUserById(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const preparedTodo: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
+
+export const getNewId = (array: { id: number }[]) => (
+  Math.max(...array.map(el => el.id)) + 1
+);
+
+export const App: React.FC = () => {
+  const [selectedUserId, setSelectedUserId] = useState(0);
+  const [todos, setTodos] = useState(preparedTodo);
+  const [title, setTitle] = useState('');
+
+  const [selectedUserIdError, setSelectedUserIdError] = useState('');
+  const [titleError, setTitleError] = useState('');
```

Comment:
let's make this states boolean
```suggestion
  const [isSelectedUserIdError, setSelectedUserIdError] = useState(false);
  const [isTitleError, setTitleError] = useState(false);
```